umbra-js/test/Umbra.test.ts

```diff
@@ -446,4 +552,97 @@ describe('Umbra class', () => {
       );
     });
   });
+
+  describe('prepareSend', () => {
+    it('should return a valid result when public keys are found', async () => {
```

Comment:
is it sufficient to just check if the properties exist?

Reply:
Yea since we have e2e tests that ensure you can withdraw sent funds, that implicitly tests that the return data here is correct, since otherwise it's hard to unit test that, so this is ok. Though I'd suggest adding that previous sentence as a comment explaining why the tests for this method are so bare

Reply:
Added the comment https://github.com/ScopeLift/umbra-protocol/commit/ddc768e7504ccb601877993210dbfa445f644277